conventions.rkt

```diff
@@ -450,7 +450,16 @@
 
 (define/record standard-formatter-map #:record all-kws
   [("if") format-if]
-  [("provide" "require" "import" "export" "link" "rename") format-require]
+  [("provide"
+    "require"
+    "import"
+    "export"
+    "link"
+    "rename"
+    "for-syntax"
+    "for-template"
+    "for-label")
```

Comment:
Thanks! I will merge this, but just curious, how do you imagine `for-meta` should be formatted?

Reply:
I think this should be allowed:

```scheme
(for-meta 2 a)
```

But if there's more than one module, it ought to be this:

```scheme
(for-meta 2
          a
          b
          c)
```